test/javascripts/components/topic-notifications-button-test.js.es6

```diff
@@ -45,3 +54,25 @@ componentTest("the header has a localized title", {
     });
   }
 });
+
+componentTest("the header has a localized title", {
+  template:
+    "{{topic-notifications-button notificationLevel=topic.details.notification_level topic=topic}}",
+
+  beforeEach() {
+    I18n.translations.en.js.topic.notifications.tracking_pm.title = `${originalTranslation} PM`;
+    this.set("topic", buildTopic(2, "private_message"));
+  },
+
+  test(assert) {
+    andThen(() => {
```

Comment:
minor nitpick here, but is this `andThen` necessary here? Unsure.

Reply:
Will merge it anyways, got to convert to async at some point so doesnt really matter